src/resources/invoice.rs

```diff
@@ -383,6 +387,18 @@ pub struct InvoiceSettingCustomField {
     pub value: String,
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct InvoiceTaxAmount {
```

Comment:
Could rename this struct and/or break it out into the shared `src/resources/types.rs` to:

- De-duplicate it between `LineItem` and `Invoice`.
- Renaming it either way to avoid the breaking change to the rust API.